src/win32/do_upgrade.ps1

```diff
@@ -101,26 +81,10 @@ write-output "$(Get-Date -format u) - Reading status file: $($status)" >> .\upgr
 If ($status -eq $null)
 {
     write-output "2" | out-file ".\upgrade\upgrade_result" -encoding ascii
-    Get-Service -Name "Wazuh" | Stop-Service
-    restore
-    write-output "$(Get-Date -format u) - Upgrade failed: Restoring." >> .\upgrade\upgrade.log
-    If ($current_process -eq "wazuh-agent")
-    {
-        .\wazuh-agent.exe install-service >> .\upgrade\upgrade.log
-    }
-    Else
-    {
-        sc.exe delete WazuhSvc -ErrorAction SilentlyContinue -Force
-        Remove-Item .\wazuh-agent.exe -ErrorAction SilentlyContinue
-        Remove-Item .\wazuh-agent.state -ErrorAction SilentlyContinue
-        .\ossec-agent.exe install-service >> .\upgrade\upgrade.log
-    }
-    Start-Service -Name "Wazuh" -ErrorAction SilentlyContinue
 }
 Else
 {
     write-output "0" | out-file ".\upgrade\upgrade_result" -encoding ascii
-    Remove-Item .\backup -recurse -ErrorAction SilentlyContinue
     write-output "$(Get-Date -format u) - Upgrade finished successfully." >> .\upgrade\upgrade.log
     $new_version = (Get-Content VERSION)
     write-output "$(Get-Date -format u) - New version: $($new_version)" >> .\upgrade\upgrade.log

```

Comment:
Can we also remove the last line (https://github.com/wazuh/wazuh/pull/9450/files?file-filters%5B%5D=.ps1#diff-2228236c7340e8be4c74b7fcf34700405182ae35ded67c9fcf309de1a076d03aR93) or why do we need it?